common/app/templates/inlineJS/blocking/config.scala.js

```diff
@@ -4,19 +4,21 @@
 @import views.support.{JavaScriptPage, CamelCase}
 @import play.api.libs.json.Json
 
+var isModernBrowser =
+    "querySelector" in document
+    && "addEventListener" in window
+    && "localStorage" in window
+    && "sessionStorage" in window
+    && "bind" in Function
+    && (
+        ("XMLHttpRequest" in window && "withCredentials" in new XMLHttpRequest())
+        || "XDomainRequest" in window
+    );
+
 window.guardian = {
-    isModernBrowser: (
-        window.shouldEnhance
-            && "querySelector" in document
-            && "addEventListener" in window
-            && "localStorage" in window
-            && "sessionStorage" in window
-            && "bind" in Function
-            && (
-                ("XMLHttpRequest" in window && "withCredentials" in new XMLHttpRequest())
-                    || "XDomainRequest" in window
-            )
-    ),
+    isModernBrowser : isModernBrowser,
+    isEnhanced:
+        window.shouldEnhance && isModernBrowser,
```

Comment:
`shouldEnhance` is defined over in `shouldEnhance.scala.js` and used once, here. It's moderately substantial, so inlining could make the file quite large. I'm happy to change things around, but that does increase the scope of this pull request.